CHANGELOG.md

```diff
@@ -94,6 +94,11 @@ It was only important with [the original open-box architecture](https://blog.sho
     - creation of entity data objects moved to factories to allow extensibility
 - [#271 - Complete refactoring of feeds functionality](https://github.com/shopsys/shopsys/pull/271)
     - for details see section shopsys/framework
+    
+### [shopsys/project-base]
+#### Fixed
+- [#195 - logout/ reports 500 for unlogged customer because of the CSRF](https://github.com/shopsys/shopsys/pull/315)
```

Comment:
please try to change the number of  issue to number of pull request with correct title
#315 Route logout/ without csrf token returns not found

Reply:
Fixed :)